TradeItIosTicketSDK2/TradeItAccountSelectionViewController.swift

```diff
@@ -23,24 +24,24 @@ class TradeItAccountSelectionViewController: CloseableViewController, TradeItAcc
         super.viewWillAppear(animated)
         
         self.promptLabel.text = promptText ?? "SELECT AN ACCOUNT FOR TRADING"
-        let enabledBrokers = TradeItSDK.linkedBrokerManager.getAllEnabledLinkedBrokers()
-        self.accountSelectionTableManager.updateLinkedBrokers(withLinkedBrokers: enabledBrokers, withSelectedLinkedBrokerAccount: selectedLinkedBrokerAccount)
-        if enabledBrokers.isEmpty {
+        let linkedBrokers = TradeItSDK.linkedBrokerManager.getAllEnabledAndActivationInProgressLinkedBrokers()
```

Comment:
This isn't a criticism of this code, but I'm starting to worry we have too many "states" to keep in mind. I will have no idea which one to use in the future. Can we think of an aggregate term for like `getAllDisplayableLinkedBrokers`? Thoughts @mitochondrion?